src/uct/api/v2/uct_v2.h

```diff
@@ -506,6 +547,30 @@ ucs_status_t uct_md_mkey_pack_v2(uct_md_h md, uct_mem_h memh,
                                  void *rkey_buffer);
 
 
+/**
+ * @ingroup UCT_MD
+ *
+ * @brief Locally attach to a remote memory.
+ * 
+ * This routine attaches a local memory handle to a memory region
+ * registered by @ref uct_md_mem_reg and packed by
+ * @ref uct_md_mem_pack_v2 by a peer to allow performing local operations
```

Comment:
remote peer ?

Reply:
imo “peer” means an another process, so it is “remote” by definition

Reply:
Fair enough. @tonycurtis any comments on this or it is clear enough ?

Reply:
Looks ok to me